projects/fsastorefrontlib/src/cms-components/seller-dashboard/seller-dashboard-list/seller-dashboard-list.component.html

```diff
@@ -1,5 +1,5 @@
 <ng-container *ngIf="customers$ | async as customers; else loading">
-  <ng-container *ngIf="customers.entries.length > 0; else noCustomers">
+  <ng-container *ngIf="customers?.entries?.length > 0; else noCustomers">
```

Comment:
The first conditional operator seems redundant since customers is already checked in the first if container?

Reply:
I don't know if it is redundant. What if we get customers, it will pass the first condition, and `customers.entries` are empty. Then we should display `noCustomers` message, right?